message/getmid.go

```diff
@@ -28,3 +29,8 @@ func RandMID() int32 {
 	}
 	return int32(uint16(binary.BigEndian.Uint32(b)))
 }
+
+// ValidateMessageID validates a message id for UDP. (0 <= mid <= 65535)
+func ValidateMessageID(mid int32) bool {
```

Comment:
I think `ValidateMID` would be more in line with the existing naming convention, but it's clear either way so I don't mind.

Reply:
Renamed 👍